elfpy/utils/outputs.py

```diff
@@ -434,6 +435,9 @@ def setup_logging(
             log_dir = os.path.join(base_folder, ".logging")
         if not os.path.exists(log_dir):  # create log_dir if necessary
             os.makedirs(log_dir)
+        # delete the log file if it exists
+        if delete_previous_logs and os.path.exists(os.path.join(log_dir, log_name)):
```

Comment:
Does this fix https://github.com/element-fi/elf-simulations/issues/254?

Reply:
ya, it actually deletes at the start of a run, which I think is better (all optional, set to off by default)
found it useful when re-running repeatedly and checking logs, to avoid having to delete it manually in between each check